frontend/www/js/omegaup/components/arena/ContestListv2.test.ts

```diff
@@ -0,0 +1,106 @@
+jest.mock('../../../../third_party/js/diff_match_patch.js');
+
+import { mount } from '@vue/test-utils';
+import type { types } from '../../api_types';
+
+import arena_ContestList from './ContestListv2.vue';
+
+describe('ContestListv2.vue', () => {
+  const daySeconds = 24 * 60 * 60 * 1000;
+  const today = new Date();
+  const yesterday = new Date(Date.now() - daySeconds);
+  const tomorrow = new Date(Date.now() + daySeconds);
```

Comment:
```suggestion
  const yesterday = new Date(today.getTime() - daySeconds);
  const tomorrow = new Date(today.getTime() + daySeconds);
```

idealmente, en las pruebas hay que hacer a lo más un `new Date()`/`Date.now()` para evitar carreras.